charts/redpanda/values.schema.json

```diff
@@ -651,23 +645,19 @@
             },
             "external": {
               "type": "object",
-              "minProperties": 1,
```

Comment:
Add a test with external set to an empty set if you change this. The chart originally depended on there being at least one external listener defined (even if external was disabled).